components/Blog.js

```diff
@@ -1,7 +1,78 @@
 const Blog = () => {
   return (
-    <div>
-      <h2>Blog Section</h2>
+    <div className="blog-main">
```

Comment:
Two of the other PRs used `className="main"` one used `className="main-container-event"` and now we've got `"blog-main"` … that seems like a good sign that you could pull this out and just make a `MainContainer` component that renders children and does the basic styling.